GPU/Common/TextureCacheCommon.cpp

```diff
@@ -787,16 +819,20 @@ bool TextureCacheCommon::AttachFramebuffer(TexCacheEntry *entry, u32 address, Vi
 			// Let's avoid using it when we know the format is wrong. May be a video/etc. updating memory.
 			// However, some games use a different format to clear the buffer.
 			if (framebuffer->last_frame_attached + 1 < gpuStats.numFlips) {
-				DetachFramebuffer(entry, address, framebuffer, channel);
+				// TODO: This is a weird outcome with unclear consequences.
```

Comment:
Thanks for catching the backwards case!

I understand the logic now. I think I want to try to rip it out and see if we still need it though... And it might be that better matching resolution can eliminate this problem anyway.

Reply:
See #6402.  Must've been for those shadows, although I think I was trying to keep some other game working...

-[Unknown]

Reply:
Oh, and better matching resolution will probably break more shadows in various games, not less....

-[Unknown]

Reply:
Well, I'm planning to take both stride and color format into account together, but not so much estimated size. So let's see.

I'm thinking about what you said about keeping track of the latest thing that happened at a memory address, maybe rather than keeping one TexCacheEntry per address/format/stride we simply want choose dynamically every time a framebuffer is bound, and scrap the whole idea of "attaching"...

Reply:
Yeah, that would probably be fine.  We'll want to consider the GE debugger too.

I think before it was partially a sequence of operations thing.  Like, we needed to know the texture wasn't a power of two for the shader ID, etc.  That's probably better now.

-[Unknown]